src/lib/db/query.js

```diff
@@ -94,17 +100,29 @@ module.exports = {
         throw new Error('Could not fetch data');
       }),
 
-  searchTimePeriodByGroup: (from, to, group) => db.one(
-      sqlBuilder({
-        'SELECT': 'count(*)::INTEGER',
-        'FROM': 'lev_audit',
-        'WHERE': [from && fromDate, to && toDate, group && searchGroup]
-      }, '\n'),
-      filterObject({ from, to, group }),
-      data => data.count
-    )
+  searchWithGroupFiltering: (from, to, group, withoutGroups) => db.one(
+    sqlBuilder({
+      'SELECT': 'count(*)::INTEGER',
+      'FROM': 'lev_audit',
+      'WHERE': [from && fromDate, to && toDate, group && searchGroup, withoutGroupsCheck(withoutGroups)]
+    }, '\n'),
+    filterObject({ from, to, group, withoutGroups }),
+    data => data.count
+  )
     .catch(e => {
       global.logger.error(`Problem retrieving a search from "${from}" to "${to}" for group "${group}"`, e);
       throw new Error('Could not fetch data');
+    }),
+
+  totalCustomerSearches: () => db.one(sqlBuilder({
+    'SELECT': 'count(*)',
+    'FROM': 'lev_audit',
+    'WHERE': ['groups <> \'{}\'::TEXT[]',
+      'NOT (groups && \'{/Monitoring,/Monitoring/Pingdom,"/Monitoring/Smoke tests",' +
+      '/LEV,/LEV/Delivery,/LEV/DSST,"/Team Delivery"}\')']
+  }), {}, data => data.count)
+    .catch(e => {
+      global.logger.error('Problem retrieving count for total customer searches', e);
+      throw new Error('Could not fetch data');
```

Comment:
Where's this whole chunk come from!?
We've changed the customer saving calculation to use *Martin's Multiplier* instead of this kind of VERY expensive SQL filtering.

Reply:
Unused remnant of previous code - removed